template/web/src/components/layout/banner/banner.tsx

```diff
@@ -5,15 +5,23 @@ import NextLink from 'next/link';
 type BannerProps = {
   pageName: string;
   pageUrl: string;
+  wip?: boolean;
 };
 
-export default function Banner({ pageName, pageUrl }: BannerProps) {
+export default function Banner({ pageName, pageUrl, wip }: BannerProps) {
   return (
     <section
-      className={clsx(
-        'flex flex-col items-center justify-between gap-6 p-6 md:flex-row md:gap-0',
-        'rounded-lg border border-zinc-400 border-opacity-20 bg-neutral-900',
-      )}
+      className={
+        wip
```

Comment:
Instead of this condition switching between these large blocks we can do this directly inside clsx because there are only slight differences between the two. This will allow us to use only one clsx call and simplify this functionality.